### PR TITLE
Applied performance indicators

### PR DESCRIPTION
**Pull request by patrickberkeley**

### CODE REVIEW

**Comment by beerlington** [comment]
LGTM but you've got a failing test

**Comment by patrickberkeley** [comment]
@beerlington please review at will

File: app/controllers/v1/applied_performance_indicators_controller.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    else
+      render_errors applied_performance_indicator
+    end
+  end
+
+  #TODO Scope to people who are on the project
+  def update
+    indicator = AppliedPerformanceIndicator.find(params[:id])
+
+    if indicator.update(create_params)
```
Comment (beerlington) [inline app/controllers/v1/applied_performance_indicators_controller.rb]: Should this be using a different set of params than the create action?

File: db/migrate/20150415180238_create_applied_performance_indicators_tiles.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class CreateAppliedPerformanceIndicatorsTiles < ActiveRecord::Migration
+  def change
+    create_join_table :applied_performance_indicators, :tiles
+    add_index :applied_performance_indicators_tiles, [:applied_performance_indicator_id, :tile_id], name: 'index_applied_perf_ind_tiles_on_applied_perf_ind_id_and_tile_id'
```
Comment (beerlington) [inline db/migrate/20150415180238_create_applied_performance_indicators_tiles.rb]: Should this be a unique index?

File: app/controllers/v1/applied_performance_indicators_controller.rb (hunk 1 above)
Comment (patrickberkeley) [inline app/controllers/v1/applied_performance_indicators_controller.rb]: :+1: Fixed in https://github.com/AgilionApps/LaunchPad-API/commit/6e3faaaf78970ebaf934780dc12cdab192b2a780.

I think this action might change a bit when I get to removing applied performance indicators from a tile.

File: db/migrate/20150415180238_create_applied_performance_indicators_tiles.rb (hunk 2 above)
Comment (patrickberkeley) [inline db/migrate/20150415180238_create_applied_performance_indicators_tiles.rb]: Fixed in https://github.com/AgilionApps/LaunchPad-API/commit/78afbe8b9d45f78874c6d524448407a1fb423f1a

**Pull request merged**